app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def last_5_comments
+    comments.last(5)
+  end
+
+  def update_comments_count
+    update(comments_counter: comments.count)
+  end
+
+  def update_likes_count
+    update(likes_counter: likes.count)
+  end
```

Comment:
- [ ] Good job adding the counter methods but it would be better if every counter is kept in their respective models. For instance, when a new comment is added you can include an active callback like `after_save` to run that method and increment the comments counter.